src/stores/api/api.store.tsx

```diff
@@ -8,22 +8,34 @@ export interface APIModel {
   classes: Record<string, any> | null;
   class: Record<string, any> | null;
   abilities: Record<string, any> | null;
+  races: Record<string, any> | null;
+  race: Record<string, any> | null;
+  allEquipment: Record<string, any> | null;
 
   generateDndClass: Action<APIModel, DnDClass>;
   generateClasses: Thunk<APIModel>;
   generateClass: Thunk<APIModel, string>;
   generateAbilities: Thunk<APIModel>;
+  generateRaces: Thunk<APIModel>;
+  generateRace: Thunk<APIModel, string>;
+  generateAllEquipment: Thunk<APIModel>;
 
   setClasses: Action<APIModel, any>;
   setClass: Action<APIModel, any>;
   setAbilities: Action<APIModel, any>;
+  setRaces: Action<APIModel, any>;
+  setRace: Action<APIModel, any>;
```

Comment:
Is there any way we can add type hint to this so we can avoid `any`?

Reply:
race and races are both objects